framework/src/test/java/viewpointtest/quals/Lost.java

```diff
@@ -0,0 +1,15 @@
+package viewpointtest.quals;
+
+import org.checkerframework.framework.qual.SubtypeOf;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
+@SubtypeOf({Top.class})
+public @interface Lost {}
```

Comment:
It is interesting that the requirement of javadoc doesn't fail on this file (and also on `Bottom.java` above)... it that check not enabled for `framework/src/test/java/`?
Can you make a separate PR before this one to ensure test sources are also checked?

Reply:
see #871